### PR TITLE
Code improvements for pulling a package from the remote repository

### PR DESCRIPTION
**Pull request by NatashaWso2**

### CODE REVIEW

File: cli/ballerina-cli-utils/src/main/ballerina/packaging_pull/packaging_pull.bal [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
 @final int MAX_INT_VALUE = 2147483647;
 @final string VERSION_REGEX = "(\\d+\\.)(\\d+\\.)(\\d+)";
+DefaultLogger logger;
+
+documentation {
+    This object denotes the default logger object used when pulling a package directly.
+}
+type DefaultLogger object {
+    public {
+        int offset = 0;
```
Comment (hemikak) [inline cli/ballerina-cli-utils/src/main/ballerina/packaging_pull/packaging_pull.bal]: offset can be private right ?

File: cli/ballerina-cli-utils/src/main/ballerina/packaging_pull/packaging_pull.bal [hunk 2]
```diff
@@ -1,11 +1,35 @@
...
 import ballerina/io;
 import ballerina/mime;
 import ballerina/http;
 
 @final int MAX_INT_VALUE = 2147483647;
 @final string VERSION_REGEX = "(\\d+\\.)(\\d+\\.)(\\d+)";
+DefaultLogger logger;
+
+documentation {
+    This object denotes the default logger object used when pulling a package directly.
```
Comment (hemikak) [inline cli/ballerina-cli-utils/src/main/ballerina/packaging_pull/packaging_pull.bal]: if offset is public, need to add it as field in the documentation block

Comment (NatashaWso2) [reply cli/ballerina-cli-utils/src/main/ballerina/packaging_pull/packaging_pull.bal]: Changed access modifier of offset to private with ea151c5

**Comment by hemikak** [comment]
PR needs to updated with the modifications for object fields access modifier change.

**Comment by NatashaWso2** [comment]
Updated the PR with the latest object syntax with cde19ca

**Pull request merged**